sql/2020/19_Compression/mbyte_saving_of_text_compression.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+#Text Based Compression Byte Savings
+SELECT 
+  _TABLE_SUFFIX AS client,
+  ROUND(CAST(JSON_EXTRACT_SCALAR(report, "$.audits.uses-text-compression.details.overallSavingsBytes")as INT64)/1024/1024) mbyte_savings,
+  COUNT(0) num_pages,
```

Comment:
Explicit `AS` and include a total field
```suggestion
  COUNT(0) AS num_pages,
  SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
```

Reply:
revised.